test/k8s-integration/driver.go

```diff
@@ -65,11 +78,19 @@ func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayNa
 	if err != nil {
 		return fmt.Errorf("failed to deploy driver: %v", err)
 	}
-
+	klog.Infof("Deploying driver")
 	// TODO (#139): wait for driver to be running
-	time.Sleep(time.Minute)
-	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", driverNamespace)
-	err = runCommand("Checking driver pods", statusCmd)
+	for i := 0; i < 3; i++ {
+		if platform == "windows" {
+			klog.Infof("Test waiting for 5 mins")
```

Comment:
Can you make this more descriptive, like "Waiting 5 minutes for the driver to start". Should we log for linux too?

Reply:
updated

Reply:
updated